content/documentation/staging/faq/distributed-tracing/turn-off-jaeger-integration.adoc

```diff
@@ -1,10 +0,0 @@
----
-title: "I see the following informative message: \"Could not fetch Jaeger info. Turning off Jaeger integration\""
-weight: 70
----
-:icons: font
-:imagesdir: /images/faq/tracing/
-
-image::jaeger_disabled.png[Jaeger Integration Disabled]
-
-Although Kiali discovered the Jaeger URL the signed up user doesn't have access to the Jaeger query endpoint. Accessing to the discovered URL (shown in the "About" modal) shouldn't be working. Once the endpoint will be accessible by the signed up user, the Jaeger integration will be enabled in Kiali.
```

Comment:
The enable param in the configuration is still there. https://github.com/kiali/kiali/blob/master/config/config.go#L155

Have we change the behaviour if If Kiali can't reach Jaeger?

Reply:
Jaeger can be disabled yes, but it's quite different from the "turning off Jaeger integration" that we had before, and which was dependent on whether URL could be reached or not. Now it's just an on/off flag pretty straightforward if I'm correct, I don't think it deserves a faq entry, right?
When Kiali can't reach jaeger, it just shows an error while fetching, without disabling anything implicitly.